### PR TITLE
Fix download URL and dependencies for py-pypar

### PR DESCRIPTION
**Pull request by hegner**
Another py-* package failure less

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pypar/package.py [hunk 1]
```diff
@@ -30,12 +30,13 @@ class PyPypar(Package):
        written in Python to run in parallel on multiple processors and
        communicate using MPI."""
     homepage = "http://code.google.com/p/pypar/"
-    url      = "https://pypar.googlecode.com/files/pypar-2.1.5_108.tgz"
+    url      = "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pypar/pypar-2.1.5_108.tgz"
 
     version('2.1.5_108', '7a1f28327d2a3b679f9455c843d850b8',
-            url='https://pypar.googlecode.com/files/pypar-2.1.5_108.tgz')
+            url = "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pypar/pypar-2.1.5_108.tgz")
```
Comment (becker33) [inline var/spack/repos/builtin/packages/py-pypar/package.py]: It looks like the URL for the version is the same as the URL. Are you sure the URL for version is necessary?

**Comment by hegner** [comment]
It circumvents a parsing error in spack that builds the URL wrongly from the version provided. Haven't debugged/fixed that part yet.

**Comment by becker33** [comment]
@hegner would this be better handled with a custom url_for_version method?

**Comment by hegner** [comment]
@becker33 - ah, totally forgot about that feature. Fixed.

**Pull request merged**